updater/spec/dependabot/file_fetcher_command_spec.rb

```diff
@@ -109,8 +109,9 @@
         allow_any_instance_of(Dependabot::Bundler::FileFetcher).
           to receive(:files).
           and_raise(exception)
+        # TODO this may be deletable since the ecosystem_versions call is now buried behind a flag...
```

Comment:
TODO before merge: try this to see if i can safely delete it... may need to first get some working ecosystems ported over to this though... like bundler, npm_and_yarn, etc.